client/src/elements/schema-editor/schema-editor-date.html

```diff
@@ -0,0 +1,6 @@
+<template class="schema-editor-input" bindable="schema, data, change, required">
```

Comment:
the bindable attribute is not necessary here as the bindable properties are defined in the view model.